<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="4">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="id">
          <property name="text">
           <string>Id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="velocidad">
          <property name="text">
           <string>Velocidad</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="distancia">
          <property name="text">
           <string>Distancia</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="mostrar_tabla_pushButton_2">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bgh">
       <attribute name="title">
        <string>Grafica</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>169</x>
          <y>90</y>
          <width>391</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dibujar">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>370</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Dibujar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="limpiar">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>370</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>171</width>
          <height>381</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="id_lineEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="origeny_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>62</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="destinox_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>88</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="origenx_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>36</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>62</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Origen de y:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>166</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Color(rgb): </string>
         </property>
        </widget>
        <widget class="QSpinBox" name="green_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>211</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>36</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Origen en x:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>114</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Destino y:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="mostrar_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>347</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="red_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>185</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="destinoy_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>114</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>140</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="blue_spinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>237</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>88</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Destino x:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="agregarinicio_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>318</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar inicio</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Velocidad:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>185</y>
           <width>19</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
        <widget class="QPushButton" name="agregarfinal_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>289</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar final</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>237</y>
           <width>20</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>211</y>
           <width>32</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Green </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>263</y>
           <width>47</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Distancia:</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="distancia_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>84</x>
           <y>263</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPlainTextEdit" name="salida_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>391</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo </string>
    </property>
    <addaction name="actionAbir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGrafo"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbir">
   <property name="text">
    <string>Abir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGrafo">
   <property name="text">
    <string>Grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
